### PR TITLE
Task 7

### PR DESCRIPTION
**Pull request by AverchenkoAA**
FE - https://dwgz1q98lr4ie.cloudfront.net/
BE with auth - https://20dpgo3rb9.execute-api.eu-west-1.amazonaws.com/dev/import?name=products.csv

FE PR - https://github.com/AverchenkoAA/nodejs-aws-fe/pull/5

Now you have to add `authorization_token` into local storage with encoded value for `averchenkoaa` user to use `/import` endpoint.

**ONLY Evaluation criteria were covered.**

Example from Postman:
image

### CODE REVIEW

**Comment by MaxPlav** [comment]
Seems one evaluation criteria is not met " import-service serverless.yaml file has authorizer configuration for the importProductsFile lambda."

**Comment by AverchenkoAA** [comment]
> Seems one evaluation criterium is not met " import-service serverless.yaml file has authorizer configuration for the importProductsFile lambda."

Fixed in the last commit.

File: src/authorization-service/serverless.yml [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+  patterns:
+    - "!*"
+    - "!*/**"
+    - dist/**
+
+functions:
+  basicAuthorizer:
+    handler: dist/handler.basicAuthorizer
+    events:
+      - http:
```
Comment (kzanton) [inline src/authorization-service/serverless.yml:36]: The http trigger not needed here, the function called from the import service

Comment (AverchenkoAA) [reply src/authorization-service/serverless.yml]: Fixed.

**Comment by AverchenkoAA** [comment]
**Cognito Auth added!**
For using `/import` endpoint you should login in cognito.

**Pull request merged**